tests/acceptance.suite.yml

```
@@ -19,15 +19,15 @@ modules:
         'browserstack.user': 'BROWSERSTACK_USERNAME'
         'browserstack.key' : 'BROWSERSTACK_ACCESS_KEY'
         'browserstack.debug': true
-        'build': 'codeception-browserstack'
+        'build': 'browserstack build'
```

```suggestion
        'build': 'browserstack build 1'
```
i think this is used in other repos..

---

done